src/components/SelectionList/BaseSelectionList.js

```diff
@@ -474,7 +472,6 @@ function BaseSelectionList({
                                     onScrollBeginDrag={onScrollBeginDrag}
                                     keyExtractor={(item) => item.keyForList}
                                     extraData={focusedIndex}
-                                    indicatorStyle={theme.white}
```

Comment:
`indicatorStyle` only accepts `"black" | "white" | "default" | undefined`, but the theme color is a hex string (#FFFFFF), so I remove it because it's invalid

The reason I notice this `indicatorStyle` is because, in BaseOptionsSelector, we set it to "white"
https://github.com/Expensify/App/blob/c0608977d90835b34382325732e755733c034240/src/components/OptionsList/BaseOptionsList.tsx#L244

and this makes the scroll bar unnoticeable for native on light mode, so we need to test it on dark mode.

Btw, `indicatorStyle` is only available for iOS. The question is, do we want to dynamically change the `indicatorStyle` based on the selected theme for iOS only?

Reply:
I think this is probably a separate issue unrelated to whether or not the scrollbar is visible based on the issue description. However, if you want to tackle this here, then the `index.ios.tsx` should be updated with the conditional check since it's an iOS-only prop.

Reply:
Any thoughts on this, @MariaHCD?

Reply:
_The recent TS migration PR corrected the value by passing "white"._

I agree that we can handle this scrollbar color in a separate issue.

Reply:
Agreed that the scrollbar color is out of scope for this PR and issue